.docker/test.ini

```diff
@@ -94,7 +94,7 @@ ckan.redis.url = redis://redis:6379
 #       Add ``resource_proxy`` to enable resorce proxying and get around the
 #       same origin policy
 # @todo:setup Cleanup the list to use only required plugins.
-ckan.plugins = stats text_view image_view recline_view datastore datarequests
+ckan.plugins = stats text_view image_view recline_view datastore data_qld_theme datarequests data_qld ytp_comments
```

Comment:
why do we need comments and theme? This looks like a circular dependency to me. Could you please clarify.

Reply:
We need theme because the theme extension contains template overrides for the datarequests core theme files - the overrides contain display functionality specific to the requirements for Data.Qld

We need the comments extension because one of the requirements for the wider Data Requests functional requirements was to utilise the same commenting engine for both datasets and datarequests